src/RabbitMQContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using RabbitMQ.Client;
+
+namespace Microsoft.Azure.WebJobs.Extensions
+{
+    public class RabbitMQContext
+    {
+        public string Hostname { get; set; }

```

Comment:
Since these properties are exposed on attribute, add RabbitMQAttribute similar to:
https://github.com/Azure/azure-webjobs-sdk-extensions/blob/dev/src/WebJobs.Extensions.CosmosDB/CosmosDBContext.cs

Reply:
Are these properties auto-resolved? I'm not sure if I need to check for null values or not when creating the context.

Reply:
These values need to supplied either via attribute or options. The validation logic would throw if both of them are not provided. When building the context, you can build it from the attribute.